packages/demos/src/server-side-rendering/feature-app.tsx

```diff
@@ -24,7 +24,8 @@ const featureAppDefinition: FeatureAppDefinition<
 
   dependencies: {
     externals: {
-      react: '^16.7.0'
+      react: '^16.7.0',
+      '@feature-hub/react': '^1.2.0'
```

Comment:
This Feature App does not use this package, does it?

Reply:
i removed it